apps/lcs/fields/index.js

```diff
@@ -106,6 +112,25 @@ module.exports = {
     ],
     className: ['govuk-label--s']
   }),
+  'extra-tenant-pob': {
+    mixin: 'input-text',
+    validate: 'required',
+    className: ['govuk-input', 'govuk-!-width-two-thirds']
+  },
+  'extra-tenant-ni-num': {
+    mixin: 'input-text',
+    validate: ['required', niNumber],
+    className: ['govuk-input', 'govuk-input govuk-!-width-one-half']
+  },
+  'extra-tenant-email': {
+    mixin: 'input-text',
+    validate: 'email',
+    className: ['govuk-input', 'govuk-!-width-two-thirds']
+  },
+  'extra-tenant-tel': {
+    mixin: 'input-text',
+    validate: ['required', 'notUrl', 'ukPhoneNumber']
```

Comment:
For international telephone number - "internationalPhoneNumber" has to be added for this field. and remove ukPhoneNumber from here.